src/lib/operators/get_table.cpp

```diff
@@ -191,9 +191,9 @@ std::shared_ptr<const Table> GetTable::_on_execute() {
         }
 
         if (current_chunk_order && current_chunk_order->first == stored_column_id) {
-          adapted_chunk_order = {
-              ColumnID{static_cast<uint16_t>(std::distance(_pruned_column_ids.begin(), pruned_column_ids_iter))},
-              current_chunk_order->second};
+          const auto pruned_column_count = std::distance(_pruned_column_ids.begin(), pruned_column_ids_iter);
```

Comment:
This is not really the number of pruned columns, but only that of those we have seen so far. `column_pruned_so_far`? Or maybe a comment? ;)

Reply:
Changed to `columns_pruned_so_far`.